docs/content/development/extensions-core/postgresql.md

```diff
@@ -47,3 +47,19 @@ Make sure to [include](../../operations/including-extensions.html) `postgresql-m
   druid.metadata.storage.connector.user=druid
   druid.metadata.storage.connector.password=diurd
   ```
+
+## Configuration
+In most cases, the configuration options map directly to the [postgres jdbc connection options](https://jdbc.postgresql.org/documentation/head/connect.html).
+
+|Property|Description|Default|Required|
+|--------|-----------|-------|--------|
+| useSSL | Enables SSL | `false` | no |
```

Comment:
The druid docs generally put the full property name in these tables, e.g. `druid.metadata.postgres.ssl.useSSL`, can you change these to that form?